science/ALPSCore/Portfile

```diff
@@ -19,13 +18,15 @@ long_description    ALPSCore contains the core libraries of the applications and
 
 homepage            http://alpscore.org
 
-checksums           rmd160  bf0fac92dfaa831379198d19e1593a241378841a \
-                    sha256  5468f73a11e603cdfe7e99ec0b5cbb55915f7fadfc04201e6c97beaa16aed4b7
+checksums           rmd160  7ef9f655f55a21658da9d7af00025d2eb7ab5a9a \
+                    sha256  01ae0c7be844d4497da96bd8b329bced3eeb4dce1101f193206e9b1c0c68068f \
+                    size 1405042
 
 compiler.blacklist  gcc-4.2
 
 depends_lib         port:boost \
-                    port:hdf5
+                    port:hdf5  \
+                    port:eigen3
```

Comment:
Can eigen3-devel also be used here?

Reply:
If both can be used, then presumably we want to specify a path rather than just a port?

Reply:
ALPSCore is tested with Eigen 3.3.4, but right now it seems to be working fine with `eigen3-devel`. As `eigen3-devel` tracks the latest development version of 3.3.x and thus is a moving target, i am not sure it's a good idea to specify a path. Maybe it is possible to have a separate variant that depends on `eigen3-devel`? How practical is it?

Reply:
Normally a path is specified. See, for example, the usage in the digikam Portfile:

`path:include/eigen3/Eigen/Eigen:eigen3`